packages/microcosm/src/data.js

```diff
@@ -117,49 +132,52 @@ export function get(object: ?Object, keyPath: *, fallback?: *) {
 
 /**
  * Merge any number of objects into a provided object.
+ * $FlowFixMe This function enumerates on variable arguments
  */
-export function merge(...args: Array<?Object>): Object {
-  let copy = null
-  let subject = null
+export function merge(): Object {
```

Comment:
The update here: instead of `...spread`'ing arguments into an array, we just enumerate over the arguments list. This prevents an array allocation and extra enumeration over `arguments`.